torch/onnx/symbolic_helper.py

```diff
@@ -482,7 +483,7 @@ def _get_tensor_rank(x: _C.Value) -> Optional[int]:
     return x_type.dim()
 
 
-def _get_tensor_sizes(x: _C.Value, allow_nonstatic=True):
+def _get_tensor_sizes(x: _C.Value, allow_nonstatic: bool = True):
```

Comment:
The polymorphic return type (Optional[List[Optional[int]]]) causes many troubles in callers when mypy does its analysis so I am skipping it